### PR TITLE
Variables: 2 tasks from 1 lesson on JS

### PR DESCRIPTION
**Pull request by limur88**

### CODE REVIEW

**Review by shnobel — DISMISSED** [review]

File: Variables/task1.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+console.log('item_6 == ' + item_6);
+console.log('item_6_type ==' + item_6_type);
+
+// 22. Создать переменную item_7 и в ней преобразовать item_6 в String.
+// 23. Создать переменную item_7_type
+// 24. Присвоить переменной item_7_type тип переменной item_7
+// 25. Вывести в консоль тип данных item_7 в виде ——  “item_7 == ”  item_7,  “item_7_type == ”  item_7_type ——  
+let item_7 = String(15);
+let item_7_type = typeof item_7;
+console.log('item_7 == ' + item_7);
```
Comment (shnobel) [inline Variables/task1.js:53]: consider to use string interpolation 

> console.log(\`item_7 == ${item_7}\`)

**Comment by limur88** [comment]
@anatoly-karpovich @VladimirLQA

**Comment by limur88** [comment]
@limur88

File: Variables/task2.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* Task2*. Решить квадратные уравнения. Переменные называть по правилам.
+Вывести в консоль ответы в виде "Ответ к уравнению 1: <корень>", "Ответ к уравнению 2: <корень> и <корень>"
+1.  x2 - 6x + 9 = 0. - один корень
+2.  x2 - 4x - 5 = 0. - два корня
+*/
+let firstDiscriminant = (-6) ** 2 - 4 * 9;
+let firstRoot = (-6 + Math.sqrt(firstDiscriminant)) / 2;
+
+
+let secondDiscriminant = (-4) ** 2 - 4 * (-5);
```
Comment (anatoly-karpovich) [inline Variables/task2.js]: I belive this-like variables should be created with const, check task 2/3

**Review by anatoly-karpovich — COMMENTED** [review]
Change let to const in task 2 and 3.

And good job!

**Review by anatoly-karpovich — DISMISSED** [review]

File: Variables/task3.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+let digit = 5;
+let firstAdditive = digit;
+let secondAdditive = String(digit) + digit;
+let thirdAdditive = secondAdditive + digit;
+let sum = digit + (+ secondAdditive) + (+ thirdAdditive);
+console.log(sum);
+
+let number = prompt('Enter digit from 1 to 9');
+let first = number;
+let second = String(number) + number;
```
Comment (anatoly-karpovich) [inline Variables/task3.js]: Prompt returns string value, so additional conversation to string is not required

File: Variables/task3.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/* Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
+n + nn + nnn, где n не перемножаются, а конкатенируются
+*/
+
+let digit = 5;
+let firstAdditive = digit;
+let secondAdditive = String(digit) + digit;
+let thirdAdditive = secondAdditive + digit;
+let sum = digit + (+ secondAdditive) + (+ thirdAdditive);
```
Comment (anatoly-karpovich) [inline Variables/task3.js]: For variables than won't be changed use const

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

**Review by anatoly-karpovich — APPROVED** [review]

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**